rcl/src/rcl/client.c

```diff
@@ -163,6 +167,7 @@ rcl_ret_t
 rcl_client_fini(rcl_client_t * client, rcl_node_t * node)
 {
   (void)node;
+  RCUTILS_LOG_DEBUG_NAMED(ROS_PACKAGE_NAME, "Finalizing client")
```

Comment:
Should this add a "client finalized" like it's done in the init function? (same for node and other node entities below)

Reply:
👍

Reply:
done in bbece9efc6e41562885394b1a07606c9fac6292e